build-scripts/get_translation_characters.py

```diff
@@ -14,6 +15,28 @@ def main():
           "el es fr hu id is it ja ko nb nl pl pt ru sr tr uk_UA zh_Hans "
           "zh_Hant > src/cldr/imgui-glyph-ranges.cpp\n")
     print("// NOLINTBEGIN(cata-static-declarations,readability-function-size)")
+    print("")
+    print("#if defined(__GNUC__) or defined(__clang__)")
```

Comment:
Can you tidy this up by moving the preprocessor nonsense into a separate function?